test/unit-tests/debugger/lldb.test.ts

```diff
@@ -0,0 +1,213 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the VS Code Swift open source project
+//
+// Copyright (c) 2021-2024 the VS Code Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of VS Code Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import * as vscode from "vscode";
+import * as util from "../../../src/utilities/utilities";
+import * as lldb from "../../../src/debugger/lldb";
+import * as fs from "fs/promises";
+import { expect } from "chai";
+import {
+    instance,
+    MockedObject,
+    mockFn,
+    mockGlobalModule,
+    mockGlobalObject,
+    mockObject,
+    MockedFunction,
+} from "../../MockUtils";
+import { SwiftToolchain } from "../../../src/toolchain/toolchain";
+import { WorkspaceContext } from "../../../src/WorkspaceContext";
+import sinon = require("sinon");
+
+suite("getLLDBLibPath Tests", () => {
+    let mockToolchain: MockedObject<SwiftToolchain>;
+    let mockFindLibLLDB: MockedFunction<(typeof lldb)["findLibLLDB"]>;
+    const mockUtil = mockGlobalModule(util);
+
+    setup(() => {
+        mockFindLibLLDB = sinon.stub();
+        mockToolchain = mockObject<SwiftToolchain>({
+            getLLDB: mockFn(),
+            swiftFolderPath: "",
+        });
+    });
+
+    teardown(() => {
+        mockFindLibLLDB.reset();
```

Comment:
Is `setup` is recreating each test, shouldn't have to reset

Reply:
Done